embeddings_test.go

```diff
@@ -108,6 +109,15 @@ func TestEmbeddingEndpoint(t *testing.T) {
 	_, err := client.CreateEmbeddings(context.Background(), EmbeddingRequest{})
```

Comment:
Yeah, this test would not catch `res` not being updated, could you please update test server here so it would return non-empty `EmbeddingResponse`?